ENGDOCS-1275 (#16965)
docker/docs: docker-hub/image-access-management.md

2. Select an organization, and navigate to the **Settings** tab
3. From the **Organizations** page select **Org Permissions**.
4. Enable Image Access Management to set the permissions for the following categories of images you can manage:
- **Organization Images**: When Image Access Management is enabled, images from your organization are always allowed. These images can be public or private created by members within your organization.
- **Docker Official Images**: A curated set of Docker repositories hosted on Hub. They provide OS repositories, best practices for Dockerfiles, drop-in solutions, and applies security updates on time.
- **Docker Verified Publisher Images**: published by Docker partners that are part of the Verified Publisher program and are qualified to be included in the developer secure supply chain. You can set permissions to **Allowed** or **Restricted**.
- **Community Images**: Images are always disabled when Image Access Management is enabled. These images are not trusted because various Docker Hub users contribute them and pose security risks.

    > **Note**
    >

>     > Image Access Management is turned off default. However, members of the `owners` team in your organization have access to all images regardless of the settings.

Reviewer: ```suggestion
    > Image Access Management is turned off by default. However, members of the `owners` team in your organization have access to all images regardless of the settings.
```

5. Select the category restrictions for your images by selecting **Allowed**.

>      Once the restrictions are applied, your members can view the Org permissions page in a read-only format.

Reviewer: ```suggestion
     Once the restrictions are applied, your members can view the organization permissions page in a read-only format.
```

6. Optional: To ensure that each org member uses images in a safe and secure environment, [enfore sign-in](../docker-hub/configure-sign-in.md).

## Verify the restrictions

>    To confirm that the restrictions are successful, have each organization member pull an image onto their local computer after signing in to Docker Desktop. If they're don't sign in, an error message is sent.

Reviewer: ```suggestion
   To confirm that the restrictions are successful, have each organization member pull an image onto their local computer after signing in to Docker Desktop. If they don't sign in, they receive an error message.
```
